feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableListItem.kt

```diff
@@ -141,8 +141,7 @@ fun TimetableListItem(
                     Spacer(modifier = Modifier.size(10.dp))
                     Text(
                         text = speaker.name,
-                        fontSize = 14.sp,
-                        fontWeight = FontWeight.Medium,
+                        style = typography().bodyMedium,
```

Comment:
We can use MaterialTheme.typography here 👀

Reply:
oh i fixed this
thanks😁

Reply:
oh sorry I'm done pushing now !